src/strike.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Rémy Hubscher. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module basic-styles/strike
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import StrikeEngine from './strikeengine';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import strikeIcon from '../theme/icons/strike.svg';
+
+/**
+ * The strike feature. It introduces the Strike button and the <kbd>Ctrl+E</kbd> keystroke.
+ *
+ * It uses the {@link module:basic-styles/strikeengine~StrikeEngine strike engine feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Strike extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ StrikeEngine ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Strike';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+		const command = editor.commands.get( 'strike' );
+		const keystroke = 'Cmd+Shift+X';
```

Comment:
It is working for me without any problems (maybe OS is blocking it on your machine?). Please use `CTRL+SHIFT+X` (uppercase and `CTRL` instead of `CMD`). It will be consistent with the way we define it in other features. Note that this will also work with command key as we internally handle that.

Reply:
I don't think CMD and CTRL are the same key, one is Ctrl the other one is Meta isn't it?K

Reply:
They are not same key, we just treat them that way, and you just need to define keystroke for `CTRL` and it will be handled correctly on Mac. Please see: http://ckeditor.github.io/editor-recommendations/general-policies/#ctrl-vs-cmd.